angelsindustries/prototypes/buildings/angels-assemblers.lua

```diff
@@ -79,7 +79,7 @@ data:extend(
       },
       crafting_categories = {"crafting"},
       crafting_speed = 0.5,
-      source_inventory_size = 1,
```

Comment:
this property is only allowed on furnace type buildings...

Reply:
then just delete it, as ingredient count is surely not what we want? We refrain from setting limits using ingredient count, rather use a different crafting category.

Reply:
these "assemblers" from industries were originally planned to use those limits, but considering they are not actually used anyway... its kind of semantics at this point.